.github/workflows/promote-reusable-workflow.yml

```diff
@@ -27,6 +27,10 @@ on:
         required: false
         default: false
         type: boolean
+      day-of-month-range:
+        description: 'The range of days in the month that this promotion should occur. e.g., "1-7" this promotion will only happen in the first week of the month.'
```

Comment:
just minor request. can you add the word "inclusive" somewhere in the description. thanks

Reply:
Good call. Done!